charts/nexus-iq/templates/deployment.yaml

```diff
@@ -56,6 +56,8 @@ spec:
           volumeMounts:
             - mountPath: /sonatype-work
               name: nxiq-pv-data
+            - mountPath: /var/log/nexus-iq-server
```

Comment:
Why /var/log/nexus-iq-server? Is /var/log some requirement of deployments?

From product perspective we like all log files to be under the same work dir, like the defaults of a normal on premise deployment.

Reply:
/var/log/nexus-iq-server is the standard place referenced by the docker docs (https://hub.docker.com/r/sonatype/nexus-iq-server), so this seems consistent. in my testing, this new mount point here allows the logs to be persisted across restart. 👍

Reply:
Yes I picked the directory from our docker docs